### DISCUSSION TITLE
Reinstall fail

### DISCUSSION THREAD

**Discussion by garmanwu** [oobabooga/text-generation-webui#4553] [Q&A]
update make webui broken, and I deleted the whole directory, and uninstall the anaconda. But it always stuck at this step.

Collecting accelerate==0.24.* (from -r temp_requirements.txt (line 1))
  Using cached https://mirrors.bfsu.edu.cn/pypi/web/packages/13/9e/ee987874058f2d93006961f6ff49e0bcb60ab9c26709ebe06bfa8707a4d8/accelerate-0.24.1-py3-none-any.whl (261 kB)
Requirement already satisfied: colorama in f:\text-generation-webui-main\installer_files\env\lib\site-packages (from -r temp_requirements.txt (line 2)) (0.4.6)
Collecting datasets (from -r temp_requirements.txt (line 3))
  Using cached https://mirrors.bfsu.edu.cn/pypi/web/packages/7c/55/b3432f43d6d7fee999bb23a547820d74c48ec540f5f7842e41aa5d8d5f3a/datasets-2.14.6-py3-none-any.whl (493 kB)
Collecting einops (from -r temp_requirements.txt (line 4))
  Using cached https://mirrors.bfsu.edu.cn/pypi/web/packages/29/0b/2d1c0ebfd092e25935b86509a9a817159212d82aa43d7fb07eca4eeff2c2/einops-0.7.0-py3-none-any.whl (44 kB)
Collecting exllamav2==0.0.7 (from -r temp_requirements.txt (line 5))
  Using cached https://mirrors.bfsu.edu.cn/pypi/web/packages/cd/e7/41222fa397cd216fc0dbf40e61f4354bfb891e9365104990ffefecba3dcb/exllamav2-0.0.7-py3-none-any.whl (90 kB)
Collecting gradio==3.50.* (from -r temp_requirements.txt (line 6))
  Using cached https://mirrors.bfsu.edu.cn/pypi/web/packages/bd/ea/ca6506e4da9b5338da3bfdd6115dc1c90ffd58c1ec50ca2792b84a7b4bdb/gradio-3.50.2-py3-none-any.whl (20.3 MB)
Collecting markdown (from -r temp_requirements.txt (line 7))
  Using cached https://mirrors.bfsu.edu.cn/pypi/web/packages/70/58/2c5a654173937d9f540a4971c569b44dcd55e5424a484d954cdaeebcf79c/Markdown-3.5.1-py3-none-any.whl (102 kB)
Requirement already satisfied: numpy==1.24.* in f:\text-generation-webui-main\installer_files\env\lib\site-packages (from -r temp_requirements.txt (line 8)) (1.24.1)
Collecting numpy==1.24.* (from -r temp_requirements.txt (line 8))
  Using cached https://mirrors.bfsu.edu.cn/pypi/web/packages/d8/ec/ebef2f7d7c28503f958f0f8b992e7ce606fb74f9e891199329d5f5f87404/numpy-1.24.4-cp311-cp311-win_amd64.whl (14.8 MB)
ERROR: Ignored the following versions that require a different python version: 1.21.2 Requires-Python >=3.7,<3.11; 1.21.3 Requires-Python >=3.7,<3.11; 1.21.4 Requires-Python >=3.7,<3.11; 1.21.5 Requires-Python >=3.7,<3.11; 1.21.6 Requires-Python >=3.7,<3.11
ERROR: Could not find a version that satisfies the requirement optimum==1.14.0 (from versions: 0.0.1, 0.1.0, 0.1.1, 0.1.2, 0.1.3, 1.0.0, 1.1.0, 1.1.1, 1.2.0, 1.2.1, 1.2.2, 1.2.3, 1.3.0, 1.4.0, 1.4.1, 1.5.0, 1.5.1, 1.5.2, 1.6.0, 1.6.1, 1.6.2, 1.6.3, 1.6.4, 1.7.0, 1.7.1, 1.7.2, 1.7.3, 1.8.0, 1.8.1, 1.8.2, 1.8.3, 1.8.4, 1.8.5, 1.8.6, 1.8.7, 1.8.8, 1.9.0, 1.9.1, 1.10.0, 1.10.1, 1.11.0, 1.11.1, 1.11.2, 1.12.0, 1.13.0, 1.13.1, 1.13.2, 1.13.3)
ERROR: No matching distribution found for optimum==1.14.0
Command '"F:\text-generation-webui-main\installer_files\conda\condabin\conda.bat" activate "F:\text-generation-webui-main\installer_files\env" >nul && python -m pip install -r temp_requirements.txt --upgrade' failed with exit status code '1'.

Exiting now.
Try running the start/update script again.

**Comment by mistrjirka** [comment]
same
